### PR TITLE
added a script to count total objects from redis

### PR DESCRIPTION
**Pull request by morparia-p**

### CODE REVIEW

File: simp_comp_webservice/service_cgi/get_poller_data.cgi [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+use strict;
+use Time::HiRes qw(usleep gettimeofday tv_interval);
+use GRNOC::WebService;
+use JSON;
+use Data::Dumper;
+use Redis;
+
+
+my $redis=Redis->new(server => "io3.bldc.grnoc.iu.edu:6380");
+
```
Comment (ajragusa) [inline simp_comp_webservice/service_cgi/get_poller_data.cgi]: should be specified in a configuration file

File: simp_comp_webservice/service_cgi/redis_tsds.pl [hunk 2]
```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/perl -w
+print "Content-type: text/html\r\n\r\n";
+
+use Redis;
+use strict;
+use GRNOC::WebService;
+use JSON;
+use Data::Dumper;
+require HTTP::Request;
```
[the remark this answers is not in the document]
Comment (morparia-p) [reply simp_comp_webservice/service_cgi/redis_tsds.pl]: Used HTTP::Request to send message to TSDS when didn't know about GRNOC::WebService::Client.

Forgot to remove it. Should I now?

Used Data::Dumper for debugging.

[the remark this answers is not in the document]
Comment (ajragusa) [reply simp_comp_webservice/service_cgi/redis_tsds.pl]: yes remove it for now... use GRNOC::WebService::Client!

**Comment by ajragusa** [comment]
pull request with #81 has all of this just going to review one

**Pull request closed without merging**